LICENSE.txt

```diff
@@ -1,15 +1,21 @@
-ISC License
+MIT License
 
-Copyright (c) 2016, GuideSmiths Ltd.
+Copyright (c) One Beyond 2022 to present.
```

Comment:
I suggest

Copyright 2016-present, GuideSmiths Ltd. Assigned to One Beyond

Reply:
Perhaps we can make it a little more clear taking into account example of other projects that have been moved between different organizations, such as [express](https://github.com/expressjs/express/blob/master/LICENSE):

```
Copyright (c) 2009-2014 TJ Holowaychuk <tj@vision-media.ca>
Copyright (c) 2013-2014 Roman Shtylman <shtylman+expressjs@gmail.com>
Copyright (c) 2014-2015 Douglas Christopher Wilson <doug@somethingdoug.com>
```

In our case would be something such as:

```
Copyright (c) 2016-2022 GuideSmiths Ltd.
Copyright (c) 2013-present One Beyond
```

And add an email if we have it.

Reply:
There's a typo in your suggestion @inigomarquinez 
```suggestion
Copyright (c) 2016-2022 GuideSmiths Ltd.
Copyright (c) 2023-present One Beyond
```

Isn't there an official way to state these changes?
I've read about both approaches of just setting the ranges or adding something like `date of copyright transferral`. So I am not sure which one is the best.